docs/RELEASE.txt

```diff
@@ -38,7 +39,7 @@ Limitations / Known Problems
       default display mode, as the data is loaded in its entirety. To view
       large datasets, it is recommended to right click on a data object and
       use the "Open As" menu item, where a subset of data to view can be
-      selected.
+      selected. This also includes files with very large numbers of objects.
```

Comment:
This seems kind of misleading to put here since the option to open only a certain number of objects is in a completely different location (User Options).

Reply:
I will revisit that statement, might just be in the wrong position. We were sent a file that was not very large data wise, but extremely large number of objects. I never got it to load on my machine.

Reply:
Sure, I would just maybe have it be a separate bullet point here under the limitations.

Reply:
Probably want to remove the original sentence from the previous bullet point.